.prettierrc

```diff
@@ -0,0 +1,7 @@
+{
+  "semi": false,
```

Comment:
should get rid of the semi-colons, they make the code mess up. Python is an example, in case you worry about the syntax, the final js still have the semi-colons ;)